cpp/src/arrow/builder.h

```diff
@@ -720,6 +720,16 @@ class ARROW_EXPORT StringBuilder : public BinaryBuilder {
 
   Status Append(const std::vector<std::string>& values,
                 const uint8_t* valid_bytes = NULLPTR);
+
+  /// \brief Append a sequence of C strings in one shot
```

Comment:
"nul-terminated C strings".

Reply:
I'm using "C string" as "nul-terminated char *".

I changed to use "nul-terminated char*" instead of "C string". It'll be more clearer.